jlm-hls/src/jlm-hls.cpp

```diff
@@ -0,0 +1,108 @@
+/*
+ * Copyright 2022 Magnus Sjalander <work@sjalander.com>
+ * See COPYING for terms of redistribution.
+ */
+
+#include <jive/view.hpp>
+
+#include <jlm/backend/llvm/jlm2llvm/jlm2llvm.hpp>
+#include <jlm/backend/llvm/rvsdg2jlm/rvsdg2jlm.hpp>
+#include <jlm/backend/hls/rvsdg2rhls/rvsdg2rhls.hpp>
+#include <jlm/backend/hls/rhls2firrtl/dot-hls.hpp>
+#include <jlm/backend/hls/rhls2firrtl/firrtl-hls.hpp>
+#include <jlm/backend/hls/rhls2firrtl/mlirgen.hpp>
+#include <jlm/backend/hls/rhls2firrtl/verilator-harness-hls.hpp>
+#include <jlm/frontend/llvm/InterProceduralGraphConversion.hpp>
+#include <jlm/frontend/llvm/LlvmModuleConversion.hpp>
+#include <jlm/ir/RvsdgModule.hpp>
+
+#include <jlm-hls/cmdline.hpp>
+
+#include <llvm/IR/LLVMContext.h>
+#include <llvm/IR/Module.h>
+#include <llvm/IRReader/IRReader.h>
+#include <llvm/Support/raw_os_ostream.h>
+#include <llvm/Support/SourceMgr.h>
+
+static void
+dump_llvm(
+	  jlm::RvsdgModule &module,
+	  std::string outfile)
+{
```

Comment:
Identation!

Reply:
Renamed to llvmToFile() and changed indentation